src/Microsoft.NetCore.Analyzers/UnitTests/Runtime/TestForNaNCorrectlyTests.Fixer.cs

```diff
@@ -353,21 +357,23 @@ public void F()
 ");
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/11741")]
-        public void CA2242_FixForComparisonWithNaNInCatchFilterClause()
+        [Fact]
+        public async Task CA2242_FixForComparisonWithNaNInCatchFilterClause()
```

Comment:
📝 This should have been re-enabled a long time ago when dotnet/roslyn#11741 was fixed